app/src/auth/components/LoginPage/index.jsx

```diff
@@ -34,8 +37,19 @@ const LoginPage = (props: Props) => {
 
 export { LoginPage }
 
-export default userIsNotAuthenticated((props: Props) => (
-    <SessionProvider>
-        <LoginPage {...props} />
-    </SessionProvider>
-))
+type Props = {
+    useEthereum?: string,
+    location: {
+        search: string,
+    },
+}
+
+export default userIsNotAuthenticated((props: Props) => {
+    const useEthereum = ['1', 'true'].includes((qs.parse(props.location.search).metamask || '').toLowerCase())
```

Comment:
We could simply check for presence, i.e. `const useEthereum = !!qs.parse(props.location.search).metamask`. Don't think we have to be too specific here.